cloudify_starlingx/utils.py

```diff
@@ -248,6 +258,26 @@ def update_deployment_site(deployment_id, site_name, rest_client):
         deployment_id, site_name)
 
 
+@with_rest_client
+def get_controller_node(deployment_id, rest_client=None):
+    """Get nodes by node type. There should be one.
+
+    :param node_type:
+    :param deployment_id:
+    :param rest_client:
+    :return:
+    """
+    nodes = []
+    for node in rest_client.nodes.list(deployment_id=deployment_id,
+                                       _includes=['properties']):
+        if CONTROLLER_TYPE in node.type_hierarchy:
+            nodes.append(node)
+    if nodes:
```

Comment:
if there should be only one controller maybe raise an error if the size of nodes >1

Reply:
The blueprint is part of the package so they will always be using our blueprint.